### PR TITLE
Add comment describing PageContainer requests state

### PR DESCRIPTION
**Pull request by kevinrobinson**
From the great conversation with @eudamonius and Kevin in person at Code for Boston.

### CODE REVIEW

File: app/assets/javascripts/student_profile/page_container.js [hunk 1]
```diff
@@ -54,6 +54,10 @@
 
         // ui
         selectedColumnKey: queryParams.column || 'interventions',
+        
+        // This holds the state of network requests, so that UI components can show that to the user.
+        // These might be null (no requests), 'pending' (request currently in-flight), and 'error' or another
+        // value if the request failed.
         requests: {
```
Comment (eudaimonious) [inline app/assets/javascripts/student_profile/page_container.js]: This is good. The value of saveNotes and discontinueService will be JSON objects where the key is the note or service id and the value is null or a string?

Comment (kevinrobinson) [inline app/assets/javascripts/student_profile/page_container.js]: Ahh yeah, thanks!  Added that in too.

**Comment by alexsoble** [comment]
Is this mergeable @kevinrobinson?

**Comment by kevinrobinson** [comment]
@alexsoble 👍

**Pull request merged**